test-framework/common/src/main/java/io/quarkus/test/common/http/TestHTTPResourceManager.java

```diff
@@ -86,7 +99,7 @@ public static void inject(Object testCase, List<Function<Class<?>, String>> endp
                         path = endpointPath;
                     }
                     String val;
-                    if (resource.ssl()) {
+                    if (resource != null && resource.ssl()) {
```

Comment:
How come this change?

Reply:
That's the one I wasn't sure about.  If there is `@TestHTTPEndpoint`, but no `@TestHTTPResource`, then `resource` is null.

I kept it like that because `@TestHTTPResource`'s SSL's default value is false.

Reply:
Okay, let's see what see CI says